### PR TITLE
Minor updates and modifications + Ops for coronastatus.ca (Closes #425)

### PR DESCRIPTION
**Pull request by mepa1363**
This PR contains some minor updates and modifications to keep up with recent developments on the master branch. It also adds the deployment instructions for Ubuntu 18.04 and closes #425 (duplicate).

### CODE REVIEW

**Review by fossecode — APPROVED** [review]

File: ops/ca/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+7. Configre DNS for your domain. You can create a "A" record set and map your instance's IP to your domain.
+
+8. Create a Nginx configuration file from the example provided in this repo. Replace `YOUR_DOMAIN` with your domain name (e.g., coronastatus.ca) first.
+
+`sudo cp nginx-site-default.example /etc/nginx/site-available/default`
+
+`sudo service nginx restart`
+
+9. Enbale HTTPS
```
Comment (fossecode) [inline ops/ca/README.md]: Enable*

Comment (mepa1363) [reply ops/ca/README.md]: Nice catch 😅

**Review by michaelmcmillan — APPROVED** [review]

**Comment by fossecode** [comment]
@mepa1363 could you resolve the conflicts and then we will get this merged?

**Comment by mepa1363** [comment]
@fossecode all done. ready to merge.

**Pull request closed without merging**